### PR TITLE
EPF-001 Admin user login

### PR DESCRIPTION
**Pull request by RAMUNIMMALA**
1.Implemented login functionality.
2.Implemented logout.

### CODE REVIEW

File: efinpro/Controllers/HomeController.cs [hunk 1]
```diff
@@ -14,6 +15,8 @@ public class HomeController : EFinProBaseController
         Users _users = null;
         String UserMailID = null;
         string Password = null;
```
Comment (RAMUNIMMALA) [inline efinpro/Controllers/HomeController.cs]: remove local variable declarations at class level

File: efinpro/Controllers/HomeController.cs [hunk 2]
```diff
@@ -14,6 +15,8 @@ public class HomeController : EFinProBaseController
         Users _users = null;
```
Comment (RAMUNIMMALA) [inline efinpro/Controllers/HomeController.cs]: provide access specifiers for all call member data

File: efinpro/Controllers/HomeController.cs [hunk 3]
```diff
@@ -35,14 +38,33 @@ public ActionResult Index(Users _userdetails)
             {
                 UserMailID = _userdetails.MailID;
                 Password = _userdetails.Password;
```
Comment (RAMUNIMMALA) [inline efinpro/Controllers/HomeController.cs]: No additional variable when there is no change in the object data

File: efinpro/Controllers/HomeController.cs (hunk 1 above)
Comment (anebhargavkrishna) [reply efinpro/Controllers/HomeController.cs]: Local variable declarations at class level was removed.

File: efinpro/Controllers/HomeController.cs (hunk 2 above)
Comment (anebhargavkrishna) [reply efinpro/Controllers/HomeController.cs]: Access specifiers for all call member data has provided.

File: efinpro/Controllers/HomeController.cs (hunk 3 above)
Comment (anebhargavkrishna) [reply efinpro/Controllers/HomeController.cs]: Removed additional variables.

File: efinpro/Controllers/HomeController.cs (hunk 1 above)
Comment (anebhargavkrishna) [reply efinpro/Controllers/HomeController.cs]: Removed unused usings @ HomeController.

**Pull request merged**